### ISSUE TITLE
Doesn't seem to work on Oculus controllers

### ISSUE THREAD

**Issue by Wildblood779** [CiarenceW/CiarencesUnbelievableModifications_H3VR#1]
Using Armswinger and Classic controls, I can't seem to figure out how to get this to work properly. The mag palming seems to work fine, but taking multiple rounds from a quickbelt doesn't work no matter which buttons are held, and the revolver eject feature doesn't seem to work either. For context, the 'grab' button by default is the trigger.

**Comment by CiarenceW** [comment]
is smart palming on? also it does work on oculus controllers, I have one.

**Comment by Wildblood779** [comment]
I'm not sure, I just use the Classic Armswinger controls - I think it might be the "force vive style grab" setting since that's the only way to get toggle grip in this game, and it rebinds the 'grab' button to the trigger (so the Oculus 'grab' button is only ever used for letting go of objects)

**Comment by CiarenceW** [comment]
yeah I didn't really test these options, I'll try and see if I can add some checks and stuff

**Comment by Wildblood779** [comment]
Ah cheers for that, appreciated :)
